core/input/input_map.cpp

```diff
@@ -43,7 +43,7 @@ int InputMap::ALL_DEVICES = -1;
 void InputMap::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("has_action", "action"), &InputMap::has_action);
 	ClassDB::bind_method(D_METHOD("get_actions"), &InputMap::_get_actions);
-	ClassDB::bind_method(D_METHOD("add_action", "action", "deadzone"), &InputMap::add_action, DEFVAL(0.5f));
+	ClassDB::bind_method(D_METHOD("add_action", "action", "deadzone"), &InputMap::add_action, DEFVAL(0.2f));
```

Comment:
You need to add a compatibility method, see [here](https://docs.godotengine.org/en/latest/contributing/development/handling_compatibility_breakages.html) for instructions

Reply:
@AThousandShips When I call `ClassDB::bind_compatibility_method`, in `_bind_compatibility_methods()` should I use the old DEFVAL value (`0.5`) or the new one? (`0.2`)

Reply:
The old one, to catch the old method